tests/test_logging_wrapper.py

```diff
@@ -0,0 +1,61 @@
+from tracemalloc import start
```

Comment:
I don't see start used, is it a superfluous import?

Reply:
And also, import order is meh

Afaik PEP says python modules emptyline external modules emptyline modules from current project